docs/new_instrument.rst

```diff
@@ -520,29 +552,29 @@ Cleans instrument for levels supplied in inst.clean_level.
 
 .. code:: python
 
-   def clean(inst):
-       return None
+   def clean(self):
+       return
 
-`inst` is a ``pysat.Instrument()`` object. ``clean`` should modify `inst`
-in-place as needed; equivalent to a custom routine.
+`inst` is a :py:class:`pysat.Instrument` object. :py:func:`clean` should modify
+`inst` in-place as needed; equivalent to a custom routine.
 
 list_remote_files
 ^^^^^^^^^^^^^^^^^
 
 Returns a list of available files on the remote server. This method is required
-for the Instrument module to support the ``download_updated_files`` method,
-which makes it trivial for users to ensure they always have the most up to date
-data. pysat developers highly encourage the development of this method, when
-possible.
+for the Instrument module to support the :py:meth:`download_updated_files`
```

Comment:
the download_updated_files isn't clickable in docs I built locally. Some are though. Should all of them link somewhere?

Reply:
I think it depends on whether or not the whole path is supplied or not.  I went for legibility in this instance, though there's probably a way to get both.